### ISSUE TITLE
Benchmarking bug?

### ISSUE THREAD

**Issue by dlh01** [mboynes/wp-utility-belt#3]
In the first iteration of the outer loop, `$j` is `0`, so the inner `while()` loop never runs:

https://github.com/mboynes/wp-utility-belt/blob/9c51ab62701ee9ae3e24668e3f5a0c919da4d395/ajax.php#L77-L81

This adds a very tiny number as the first item in `$times['a']` and (`$times['b']`).

Is this intentional? Something like "warm-up" period?

**Comment by mboynes** [comment]
have you confirmed the while loop doesn't run the first iteration? since the `for` loop uses `++$j` instead of `$j++`, I would _think_ it should run. if it doesn't run, I think you're right that it's a bug and was _not_ deliberate

**Comment by dlh01** [comment]
I'm fairly confident it doesn't. Here's a shot of Xdebug where `$j === 0`:

<img width="791" alt="Screen Shot 2020-04-10 at 7 43 54 PM" src="https://user-images.githubusercontent.com/697432/79029730-b19c2300-7b63-11ea-85b0-0ccb630dc170.png">

and here's the value of `$times` during a benchmark with two iterations of `sleep( 1 )`, which shows the two items for both `a` and `b`, the first being very small.

<img width="717" alt="Screen Shot 2020-04-10 at 7 42 17 PM" src="https://user-images.githubusercontent.com/697432/79029732-b234b980-7b63-11ea-9a33-d535da300390.png">
